app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormTemplate.scala

```diff
@@ -57,7 +60,7 @@ object FormTemplate {
   import JsonUtils._
 
   private val reads = Reads[FormTemplate] { json =>
-    Json.reads[FormTemplate].reads(json)
+    Jsonx.formatCaseClass[FormTemplate].reads(json)
```

Comment:
Why are you using Jsonx?

Reply:
It allows us to read JSON format for case class with more than 22 members.